tests/components/roborock/test_config_flow.py

```diff
@@ -346,15 +351,7 @@ async def test_discovery_not_setup(
     with (
         patch("homeassistant.components.roborock.async_setup_entry", return_value=True),
     ):
-        result = await hass.config_entries.flow.async_init(
-            DOMAIN,
-            context={"source": config_entries.SOURCE_DHCP},
-            data=DhcpServiceInfo(
-                ip=NETWORK_INFO.ip,
-                macaddress=NETWORK_INFO.mac.replace(":", ""),
```

Comment:
The MAC here was already lower-case and correctly stripped of colons, but now the Mock class does it automatically